### DISCUSSION TITLE
Is there any reason @op_chain_block_time is a constant?

### DISCUSSION THREAD

**Discussion by ohbyeongmin** [blockscout/blockscout#10296] [Q&A] [answered]
Hi.
I am using customised L2 block times when creating an op-stack based network.
If i use span batch, i will get incorrect information because the value of [@op_chain_block_time](https://github.com/blockscout/blockscout/blob/acd3940b5edbe7abc794296b6113949f442961a9/apps/indexer/lib/indexer/fetcher/optimism/txn_batch.ex#L34) in the indexer is fixed.

Do you plan to give it a default value of 2 and use it as an environment variable?

**Comment by varasev** [comment, marked as answer]
@ohbyeongmin Hi, we added `INDEXER_OPTIMISM_BLOCK_DURATION` env variable in v6.10.0, so now you can explicitly define the block duration for your chain. Please, see https://docs.blockscout.com/setup/env-variables/backend-envs-chain-specific#optimism-rollup-management for the short description
